lib/mpool/include/hse/mpool/mpool_structs.h

```diff
@@ -33,6 +34,10 @@
 #define MPOOL_MBLOCK_PREALLOC   (1u << 0)   /* advisory */
 #define MPOOL_MBLOCK_PUNCH_HOLE (1u << 1)
 
+#define MPOOL_RA_PAGES_MIN      0
+#define MPOOL_RA_PAGES_MAX      ((1024 * 1024) / PAGE_SIZE)
+#define MPOOL_RA_PAGES_DFLT     ((128 * 1024) / PAGE_SIZE)
```

Comment:
These are only used in `mclass.c` and should probably be defined local to that file.